yellowbrick/model_selection/CVScores.py

```diff
@@ -0,0 +1,151 @@
+        in order to make most visualizations work.
+
+        These parameters can be influenced later on in the visualization
+        process, but can and should be set as early as possible.
+
+        Parameters
+        ---------- 
+
+        model : a scikit-learn estimator
+                An object that implements ``fit`` and ``predict``, can be a
+                classifier, regressor, or clusterer so long as there is also a valid
+                associated scoring metric.
+                Note that the object is cloned for each validation.
+
+        ax : matplotlib.Axes object, optional
+                The axes object to plot the figure on.
+
+        cv : int, cross-validation generator or an iterable, optional
+                Determines the cross-validation splitting strategy.
+                Possible inputs for cv are:
+                    - None, to use the default 3-fold cross-validation,
+                    - integer, to specify the number of folds.
+                    - An object to be used as a cross-validation generator.
+                    - An iterable yielding train/test splits.
+
+                see the scikit-learn
+                `cross-validation guide <http://scikit-learn.org/stable/modules/cross_validation.html>`_
+                for more information on the possible strategies that can be used here.
+
+        scoring : string, callable or None, optional, default: None
+                A string or scorer callable object / function with signature
+                ``scorer(estimator, X, y)``. 
+        
+                See scikit-learn model evaluation documentation for names of possible metrics.
+
+        kwargs : dict
+                Keyword arguments that are passed to the base class and may influence
+                the visualization as defined in other Visualizers.            
+
+        """
+        super(CVScores, self).__init__(model, ax=ax, **kwargs)
+    
+        self.cv = cv
+        self.scoring = scoring
+
+    def fit(self, X, y, **kwargs):
+        """
+        Fits the learning curve with the wrapped model to the specified data.
+        Draws training and test score curves and saves the scores to the
+        estimator.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+                Training vector, where n_samples is the number of samples and
+                n_features is the number of features.
+
+        y : array-like, shape (n_samples) or (n_samples, n_features), optional
+                Target relative to X for classification or regression;
+                None for unsupervised learning.
+
+        Returns
+        -------
+        self : instance
+
+        """
+
+        self.cv_scores_ = cross_val_score(self.estimator, X, y, cv=self.cv, scoring=self.scoring)
+        self.cv_scores_mean_ = np.mean(self.cv_scores_)
+
+        self.draw()
+        return self
+
+    def draw(self, **kwargs):
```

Comment:
I really like how you got this to work. great job!  The code just needs a bit better commenting.